apps/vs-agent/package.json

```diff
@@ -81,6 +82,7 @@
     "axios": "^1.7.9",
     "bull": "^4.16.2",
     "credo-ts-didweb-anoncreds": "^0.0.1-alpha.10",
+    "didwebvh-ts": "^2.4.1",
```

Comment:
There is already a 2.5.3 version. Not sure if it fixes issues on node.js but it's probably worthy to use this latest version.